Purchases/Purchasing/OfferingsManager.swift

```diff
@@ -127,17 +128,17 @@ private extension OfferingsManager {
         return Set(productIdenfitiersArray)
     }
 
-    func logMissingProductsIfAppropriate(products: [String: SKProduct], offeringsData: [String: Any]) {
-        guard !products.isEmpty,
-              !offeringsData.isEmpty else {
+    func logMissingProductIDsIfAppropriate(productsFromStore: [String: SKProduct],
+                                           productIDsFromRC: Set<String>) {
+        guard !productIDsFromRC.isEmpty else {
```

Comment:
suuuper minor, FFTI: I'd maybe call these `productIDsFromBackend`, since everything is from RC

Reply:
my thinking was that the products are from storekit, but yeah i still think that's more clear.